test/functional/lua/ui_event_spec.lua

```diff
@@ -390,6 +390,13 @@ describe('vim.ui_attach', function()
       9              bufname(        {12: }        |
       Excommand:call bufadd^(                  |
     ]])
+    -- No crash after _cmdline_offset window is closed #35584.
+    exec_lua(function()
+      vim.ui_detach(_G.ns)
```

Comment:
Though completely evil, adding a `vim.cmd.tabnew()` here will still crash because `cmdline_win = NULL` is not run for `win_close_othertab`. Maybe better to move that to `win_free`?

Reply:
I have no idea when to open new tab to reproduce similar crash.

Reply:
Caught!

Reply:
Not caught after [24e8333](https://github.com/neovim/neovim/pull/35617/commits/24e83337dc735b6f1289d73b4ebe759a88cf65c3).